flask_iiif/restful.py

```diff
@@ -183,12 +184,13 @@ def get(self, version, uuid, region, size, rotation, quality,
         # Trigger event after proccess the api request
         iiif_after_process_request.send(self, **api_after_request_parameters)
 
-        # Built the after request parameters
-        api_after_request_parameters = dict(
-            mimetype=mimetype,
-            image=to_serve
-        )
+        dlparam = secure_filename(request.args.get("dl", ''))
```

Comment:
My understanding is that we should use directly the filename as specified in `dl` argument.

```python
send_file_kwargs = {'mimetype': mimetype}
filename = secure_filename(request.args.get('dl'))
if filename:
    send_file_kwargs['as_attachement'] = True
    send_file_kwargs['attachment_filename'] = filename

return send_file(to_serve, **send_file_kwargs)
```